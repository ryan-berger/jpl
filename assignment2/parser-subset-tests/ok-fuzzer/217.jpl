print ""
read image "K3kXYB/" to d8L_
