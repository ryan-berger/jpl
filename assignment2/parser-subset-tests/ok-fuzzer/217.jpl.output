(PrintCmd "")
(ReadImageCmd "K3kXYB/" (VarArgument d8L_))
Compilation succeeded
