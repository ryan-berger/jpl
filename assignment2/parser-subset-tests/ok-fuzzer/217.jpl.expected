(PrintCmd "")
(ReadImageCmd "K3kXYB/" (VarArgument d8L_))
